src/containers/SubmissionsPanel/index.js

```diff
@@ -69,11 +69,15 @@ class SubmissionsPanelComponent extends React.Component {
       <React.Fragment>
         <Table>
           <Table.Header>
-            <Table.HeaderCell flexGrow={4}>Bounty title</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={3}>Fulfiller</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Submission date</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Status</Table.HeaderCell>
-            <Table.HeaderCell flexGrow={2}>Payment amount</Table.HeaderCell>
+            <Table.HeaderCell flexBasis="40%">Bounty title</Table.HeaderCell>
```

Comment:
I love it :) % flexbasis here

Reply:
@corwinharrell  my only Q here that confuses me - is some of these don't need to grow vs. the title as an example.